s3transfer/download.py

```diff
@@ -191,8 +191,8 @@ def _get_temp_fileobj(self):
         return f
 
     def _get_fileobj(self, filename):
-        f = DeferredOpenFile(filename, mode='wb')
-        f.OPEN_METHOD = self._osutil.open
+        f = DeferredOpenFile(
+            filename, mode='wb', open_method=self._osutil.open)
```

Comment:
I would prefer this be called `open_function`, as that most accurately describes the arg (it's also how what you describe it in your docstring). Plus the default value is the `open` function.